src/envoy/mixer/grpc_transport.cc

```diff
@@ -92,9 +92,10 @@ void GrpcTransport<RequestType, ResponseType>::onSuccess(
 
 template <class RequestType, class ResponseType>
 void GrpcTransport<RequestType, ResponseType>::onFailure(
-    Grpc::Status::GrpcStatus status) {
-  ENVOY_LOG(debug, "{} failed with code: {}", descriptor().name(), status);
```

Comment:
let's do that in separate PR, `const RequestType&` has different lifecycle so it might cause issue.